doc/sphinx/changes.rst

```diff
@@ -6,6 +6,377 @@ Recent changes
+  already compatible with a double implication or setoid equality for
+  the conclusion head symbol `R2`. But a further step consists in
+  tagging the generated subgoal `R1 (f1 i) (?f2 i)` to protect it from
+  unwanted evar instantiation, and get `Under_rel _ R1 (f1 i) (?f2 i)`
+  that is displayed as ``'Under[ f1 i ]``. In Coq 8.10, this second
+  (convenience) step was only performed when `R1` was Leibniz' `eq` or
+  `iff`. Now, it is also performed for any relation `R1` which has a
+  ``RewriteRelation`` instance (a `RelationClasses.Reflexive` instance
+  being also needed so :tacn:`over` can discharge the ``'Under[ _ ]``
+  goal by instantiating the hidden evar.) Also, it is now possible to
+  manipulate `Under_rel _ R1 (f1 i) (?f2 i)` subgoals directly if `R1`
+  is a `PreOrder` relation or so, thanks to extra instances proving
+  that `Under_rel` preserves the properties of the `R1` relation.
+  These two features generalizing support for setoid-like relations is
+  enabled as soon as we do both ``Require Import ssreflect.`` and
+  ``Require Setoid.`` Finally, a rewrite rule ``UnderE`` has been
+  added if one wants to "unprotect" the evar, and instantiate it
+  manually with another rule than reflexivity (i.e., without using the
+  :tacn:`over` tactic nor the ``over`` rewrite rule). See also Section
+  :ref:`under_ssr` (`#10022 <https://github.com/coq/coq/pull/10022>`_,
+  by Erik Martin-Dorel, with suggestions and review by Enrico Tassi
+  and Cyril Cohen).
+- **Added.**
+  A :g:`void` notation for the standard library empty type (:g:`Empty_set`)
+  (`#10932 <https://github.com/coq/coq/pull/10932>`_, by Arthur Azevedo de
+  Amorim).
+
+**Commands and options**
+
+- **Removed.**
+  Deprecated flag `Refine Instance Mode`
+  (`#9530 <https://github.com/coq/coq/pull/9530>`_, fixes
+  `#3632 <https://github.com/coq/coq/issues/3632>`_, `#3890
+  <https://github.com/coq/coq/issues/3890>`_ and `#4638
+  <https://github.com/coq/coq/issues/4638>`_
+  by Maxime Dénès, review by Gaëtan Gilbert).
+- **Removed.**
+  Undocumented :n:`Instance : !@type` syntax
+  (`#10185 <https://github.com/coq/coq/pull/10185>`_, by Gaëtan Gilbert).
+- **Removed.**
+  Deprecated ``Show Script`` command
+  (`#10277 <https://github.com/coq/coq/pull/10277>`_, by Gaëtan Gilbert).
+- **Added.**
+  Unsafe commands to enable/disable guard checking, positivity checking
+  and universes checking (providing a local `-type-in-type`).
+  See :ref:`controlling-typing-flags`
+  (`#10291 <https://github.com/coq/coq/pull/10291>`_ by Simon Boulier).
+- **Fixed.**
+  Two bugs in :cmd:`Export`. This can have an impact on the behavior of the
+  :cmd:`Import` command on libraries. `Import A` when `A` imports `B` which exports
+  `C` was importing `C`, whereas :cmd:`Import` is not transitive. Also, after
+  `Import A B`, the import of `B` was sometimes incomplete.
+  (`#10476 <https://github.com/coq/coq/pull/10476>`_, by Maxime Dénès).
```

Comment:
Despite being a fix, this should probably be part of the highlights, and we could add a `.. warning::` to signal that this bug fix can break a lot of scripts.